src/dev-server/middlewares.js

```diff
@@ -6,7 +6,7 @@ export function lambdaMiddleware(cdnBaseUrl) {
   const proxy = httpProxy.createProxyServer({ target: cdnBaseUrl });
 
   return async (req, res, next) => {
-    if ((/^\/rooms\/.+$/).test(req.url)) {
+    if ((/^\/(room-media|document-input-media)\/.+$/).test(req.url)) {
```

Comment:
This should have stopped working locally some time ago, we have to check if this code actually works